client/web/src/hooks/node-data.ts

```diff
@@ -114,8 +114,21 @@ export default function useNodeData() {
   )
 
   useEffect(
-    fetchNodeData,
-    // Initial data load.
+    () => {
+      // Initial data load.
+      fetchNodeData()
+
+      // Refresh on browser tab focus.
+      const onVisibilityChange = () => {
+        document.hidden && fetchNodeData()
```

Comment:
MDN [mentions](https://developer.mozilla.org/en-US/docs/Web/API/Document/hidden) that `document.hidden` is historical, and we should use `document.visibilityState` instead.  Also, isn't this logic backwards?  We want to fetch node data when the document *isn't* hidden?  Or now, according to [these samples](https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilitychange_event#examples), when `document.visibilityState === "visible"` ?

Reply:
whoops yep! good catch